### PR TITLE
Fix/frontend hotfixes

### PR DESCRIPTION
**Pull request by leonkho**
- Used custom breakpoint for 4 vault cards in a row
- Adjusted vault card gradient
- Added mono logos to determine vault card's respective chain
- Removed Your Position details when not connected to a chain
- Standardized wAVAX and sAVAX theme on bridge UI

### CODE REVIEW

File: shared/src/components/Product/Theta/YieldCard.tsx [hunk 1]
```diff
@@ -332,30 +338,61 @@ const YieldCard: React.FC<YieldCardProps> = ({
...
       return (
-        <div className="d-flex w-100 justify-content-center">
-          <SecondaryText fontSize={12} color={colors.primaryText}>
-            Funds ready for migration to V2
-          </SecondaryText>
-        </div>
+        <ModalContentExtra style={{ paddingTop: 14 + 16, paddingBottom: 14 }}>
+          <div className="d-flex w-100 justify-content-center">
+            <SecondaryText fontSize={12} color={colors.primaryText}>
+              Funds ready for migration to V2
```
Comment (fqlx) [inline shared/src/components/Product/Theta/YieldCard.tsx:344]: I know this is unrelated but in the future, please add text to the `translation.json` so we can work down the tech debt

File: shared/src/components/Product/Theta/YieldCard.tsx [hunk 2]
```diff
@@ -332,30 +338,61 @@ const YieldCard: React.FC<YieldCardProps> = ({
...
+    else if (isAvaxVault(vault)) logo = <AVAXMonoLogo />;
+    else if (isSolanaVault(vault)) logo = <SOLMonoLogo />;
+
+    if (logo) {
+      return (
+        <ProductTag className="p-1" color={color}>
+          <Subtitle>{logo}</Subtitle>
+        </ProductTag>
+      );
+    } else {
```
Comment (fqlx) [inline shared/src/components/Product/Theta/YieldCard.tsx:392]: nit: you could remove the else

File: webapp/src/components/Vault/VaultActionsForm/v2/VaultV2DepositWithdrawForm.tsx [hunk 3]
```diff
@@ -447,7 +447,11 @@ const VaultV2DepositWithdrawForm: React.FC<VaultV2DepositWithdrawFormProps> = ({
           target="_blank"
           rel="noreferrer noopener"
         >
-          <BridgeButton color={getAssetColor(vaultActionForm.depositAsset)}>
+          <BridgeButton
+            color={getAssetColor(
+              vaultOption === "rsAVAX-THETA" ? "sAVAX" : "WAVAX"
```
Comment (fqlx) [inline webapp/src/components/Vault/VaultActionsForm/v2/VaultV2DepositWithdrawForm.tsx:452]: Could we move this inlined conditional into the `getAssetColor()` function?  Put the hack in one spot

**Review by fqlx — DISMISSED** [review]

**Review by cjinghong — DISMISSED** [review]

**Review by fqlx — APPROVED** [review]
thanks!

**Review by cjinghong — APPROVED** [review]

**Pull request merged**